assignments/function-conversion.js

```diff
@@ -1,24 +1,18 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction = () => {
+console.log("Function was invoked!");
+}
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = (param) => param;
```

Comment:
Nice use of shorthand here! You can even drop the  ( ) on a function that only has a single parameter